zhaquirks/sonoff/rangeextender.py

```diff
@@ -0,0 +1,98 @@
+"""Device handler for Sonoff / Itead Router."""
+
+from zigpy.profiles import zha
+from zigpy.quirks import CustomDevice
+from zigpy.zcl.clusters.general import (
+    Basic,
+    Groups,
+    Identify,
+    LevelControl,
+    OnOff,
+    Scenes,
+)
+from zigpy.zcl.clusters.lighting import Color
+from zigpy.zcl.clusters.lightlink import LightLink
+
+from zhaquirks.const import (
+    DEVICE_TYPE,
+    ENDPOINTS,
+    INPUT_CLUSTERS,
+    MODELS_INFO,
+    OUTPUT_CLUSTERS,
+    PROFILE_ID,
+)
+
+
+class IteadRangeExtender(CustomDevice):
+    """Itead dongle with range extender firmware devices."""
```

Comment:
```suggestion
    """Sonoff-E dongle with range extender firmware."""
```

Reply:
All Sonoff EFR32MG21 coordinators is having the same pins / pads and can using both firmware.
This is for the Itead firmware and is working on Sonoff, eWeLink and Itead hardware.
Then like quirking the XPS firmware we calling it Sonoff.

Reply:
Hmm, okay. Suggested Dongle-E because that's how the firmware identifies itself but this is fine too.